src/process/process.lisp

```diff
@@ -84,6 +84,19 @@ IMPORTANT NOTE: Use #'SPAWN-PROCESS to generate a new PROCESS object."))
   (setf (process-command-stack process)
         (append commands (process-command-stack process))))
 
+(defun mailbox-size (process)
+  "Calculates the size of the malibox message queue for `PROCESS'."
+  (with-slots (channel secret) (process-key process)
+    (when (eq secret (gethash channel (courier-secrets *local-courier*)))
+      (q-len (gethash channel (courier-inboxes *local-courier*))))))
+
+(defun clear-inbox (process)
+  "Empties out the message queue in `PROCESS's inbox at `*LOCAL-COURIER*'"
+  (with-slots (channel secret) (process-key process)
+      (when (eq secret (gethash channel (courier-secrets *local-courier*)))
```

Comment:
in both of these, consider `assert` rather than `when`

Reply:
good call